app/build.gradle

```diff
@@ -183,6 +184,7 @@ dependencies {
 }
 
 task verifyWebViewCount(type: VerifyWebViewCount)
+task validateAndroidAppReleaseConfiguration(type: ValidateAndroidAppReleaseConfiguration)
```

Comment:
Optional, but I wonder if clarity would be improved if tasks taken from our plugin mentioned that somewhere in their names?  Otherwise this blends in a bit with magic taken from other plugins.

Reply:
Do you mean the tasks defined in the plugin should be namespaced for better clarity? e.g. `task ...(type: MozillaValidateAndroidAppReleaseConfiguration)` That seems reasonable to me (though most other gradle plugins don't seem to do this). That'd need to get updated in the plugin though so there's nothing to do here until the next plugin version.

Or did you mean that the tasks we define locally, e.g. `./gradlew validateAndroidAppReleaseConfiguration`, should mention that it's from our plugin? e.g. `./gradlew mozillaValidateAndroidAppReleaseConfiguration` I'd lean against this – tasks should be named by what they do rather than where they come from – but I can see the use case where validateAndroidAppReleaseConfiguration breaks our build and no one knows to look internally because they haven't heard of it before and aren't familiar enough with gradle to know where it came from.

Will wait for a response before deciding the next course of action.

Reply:
I guess it could be `./gradlew validateMozillaAndroidAppReleaseConfiguration`. And this namespacing rule doesn't need to apply to all tasks either.

Reply:
I was suggesting the former (`task ...(type: MozillaValidateAndroidAppReleaseConfiguration)`).  I think that should be fine.  If the task fails people will jump to the source and see that it's homegrown, so I'm not too worried about how we define it locally (although I wouldn't be opposed to updating that either).

Reply:
yolo: helpfulness > conventions. :)

Reply:
Translation: I'll do both.

Reply:
Filed https://github.com/mozilla-mobile/android-automation-tools/issues/44 to update the task name, will do the other one locally.